Makefile

```diff
@@ -109,18 +109,6 @@ toxu: export CPPFLAGS = $(_CPPFLAGS)
 toxu: $(UNITTEST_TARGETS)
 	$(TOX) -l | grep -e-unit- | xargs -n1 $(TOX) -e
 
-# Run just the tests selected in the 'cur' tox environment
-toxc: export CPPFLAGS = $(_CPPFLAGS)
-# When iterating, don't bother with slightly long lines
-toxc: PEP8_MAX_LINE := --max-line-length=120
-toxc: $(UNITTEST_TARGETS) $(KAFKA_RUN)
-	KAFKA_VERSION=$(KAFKA_VER) $(TOX) -e cur
-
-# Run the just the tests selected in tox_cur.ini until they fail
-toxrc: export CPPFLAGS = $(_CPPFLAGS)
-toxrc: $(UNITTEST_TARGETS) $(KAFKA_RUN)
```

Comment:
I actually found this target pretty helpful when trying to track down Heisenbugs (sp?), but YMMV...

Reply:
I removed it because I broke it a while back: there is no ``cur`` Tox environment anymore. Personally I use this bit of shell to re-run the tests as I edit:

```sh
TEST_OBJECT=afkak.test
# TEST_OBJECT=afkak.test.test_client:TestKafkaClient
# TEST_OBJECT=afkak.test.test_protocol
# TEST_OBJECT=afkak.test.test_brokerclient:BrokerClientTests
export TEST_OBJECT
{
    echo setup.py
    find afkak -name '*.py'
    find docs -name '*.rst'
} | AFKAK_TWISTED_DEBUG=y entr -c sh -c '
tox -e py37-unit -- --nocapture $TEST_OBJECT &&
    tox -e py27-unit -- --nocapture $TEST_OBJECT &&
    tox -e py37-lint &&
    tox -e docs
'
```